build_children.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#Gets the latest Master build
+GetMasterBuild () {
+
+    BUILD=$(curl -s 'https://api.travis-ci.org/repos/RackHD/'$1'/branches/master' | grep -o '{"branch":{"id":[0-9]*,' | grep -o '[0-9]' | tr -d '\n')
+
+    echo $BUILD
+}
+
+# Get last child project build number
+BUILD_NUM_TASKGRAPH=$(GetMasterBuild on-taskgraph)
```

Comment:
Ok, then I will have on-tasks build it instead of on-core. As on-core builds on-tasks.